<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NetChatServer</string>
  </property>
  <widget class="QPushButton" name="pbSend">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>520</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send to all</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lwClients">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>211</width>
     <height>341</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pteMessage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbToAll">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>520</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>To all</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Type a message:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Configuration</string>
   </property>
   <widget class="QPushButton" name="pbStop">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbStart">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbPort">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>65530</number>
    </property>
    <property name="value">
     <number>1616</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leHost">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Authorized users:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbAutostart">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Autostart ChatServer</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="teLogArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>461</width>
     <height>341</height>
    </rect>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leHost</tabstop>
  <tabstop>sbPort</tabstop>
  <tabstop>pbStart</tabstop>
  <tabstop>pbStop</tabstop>
  <tabstop>pteMessage</tabstop>
  <tabstop>pbSend</tabstop>
  <tabstop>cbToAll</tabstop>
  <tabstop>lwClients</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
